### ISSUE TITLE
Ensure 2.0 is as easy to install as 1.x (aka update release docs)

### ISSUE THREAD

**Issue by jodh-intel** [kata-containers/kata-containers#538]
Kata 1.x can be installed with docker using a single command:

- https://github.com/kata-containers/documentation/blob/master/install/installing-with-kata-manager.md#full-installation

This is a compelling feature, particularly for new users as it gives a working environment in minutes.

If we drop docker support, we need to rewrite the [2.0 install docs](https://github.com/kata-containers/kata-containers/tree/2.0-dev/docs/install) to ensure it's as easy as 1.x to install 2.0 (with docker/containerd/podman/whatever).

**Comment by jodh-intel** [comment]
If we [re-add the OCI CLI for Kata 2.0](https://github.com/kata-containers/kata-containers/issues/722), this issue can be closed after we've tested the CI for 2.0.

However, if this doesn't happen, the simplest alternative to docker appears to be a containerd install guide.

- I've created https://github.com/kata-containers/kata-containers/pull/739

  Still being worked on.

- After that lands, we'll need to make changes to https://github.com/kata-containers/tests/tree/master/cmd/kata-manager

  It will need to be able to install a containerd-based system but also work with 2.0 as well as 1.x.
